roles/prometheus/vars/main.yml

```diff
@@ -1,2 +1,2 @@
 ---
-prometheus_role_version: 1.2.2
\ No newline at end of file
+prometheus_role_version: 1.2.3
```

Comment:
Why did you updated prometheus role version while the role updated was Grafana ?

Reply:
I was working on both roles simultaneously and made a confusion.